### DISCUSSION TITLE
swiper container flex issues

### DISCUSSION THREAD

**Discussion by JyotiRSharma** [nolimits4web/swiper#7589] [Q&A]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/p/sandbox/sleepy-silence-td45kk

### Bug description

When using `display: flex;` property, in Edge & Firefox, the swiperContainer width becomes so big that it goes out of the window.
But in Chrome is looks perfectly fine.
**Chome:**
![image](https://github.com/nolimits4web/swiper/assets/28672278/1329ef3a-c5e5-422f-934a-5ea399234ef2)
**Edge:**
![image](https://github.com/nolimits4web/swiper/assets/28672278/b03fd7bb-f2ca-4087-8daa-8ac8368d062d)
**Firefox:**
![image](https://github.com/nolimits4web/swiper/assets/28672278/03d6b001-3f6f-4da3-b358-eae3a09e41be)

Check it out here:
https://td45kk.csb.app/

### Expected Behavior

Swiperslides should behave exactly how it behaves on chrome across all browsers

### Actual Behavior

Swiperslides displays expected results in chrome but not on other browsers like edge and firefox.

### Swiper version

10.0.0

### Platform/Target and Browser Versions

Windows 10 Chrome 125.0.6422.113 Firefox 125.0.6422.113 Edge 126.0.2592.68

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [X] I'm willing to open a PR

**Comment by DriveSoft** [comment]
The same, on Google Chrome Version 126.0.6478.126
